app/controllers/follow_requests_controller.rb

```diff
@@ -67,4 +68,11 @@ def set_follow_request
     def follow_request_params
       params.require(:follow_request).permit(:recipient_id, :sender_id, :status)
     end
+
+    def ensure_current_user_is_owner
```

Comment:
Looks like its working for both recipient and sender, but could also create this same logic for the `@follow_request.recipient` to check for update